Source/LibationCli/Options/_ProcessableOptionsBase.cs

```diff
@@ -32,13 +43,28 @@ protected static TProcessable CreateProcessable<TProcessable>(EventHandler<Libra
 				}
 			};
 
+			strProc.StreamingTimeRemaining += (_, e) => progressBar.RemainingTime = e;
+			strProc.StreamingProgressChanged += (_, e) => progressBar.Progress = e.ProgressPercentage;
+
 			return strProc;
 		}
 
-		protected static async Task RunAsync(Processable Processable)
+		protected async Task RunAsync(Processable Processable)
```

Comment:
With a little refactoring, you might be able to speed up some of these loops with await foreach or Task.WhenAll.

Reply:
I've thought about adding a multithread feature to the GUI to allow users to run multiple processables simultaneously. It's something I'm interested in doing, so I'll save multithreading the cli until then. Please remind me then if I forget.